fees/suilend/index.ts

```diff
@@ -37,7 +38,8 @@ const fetchSuilendStats = async ({ endTimestamp }: FetchOptions) => {
 
   const dailyRevenue = stats.borrowFees +
     stats.protocolFees +
-    stats.liquidationProtocolFees;
+    stats.liquidationProtocolFees
+    + stats.stakingRevenue;
```

Comment:
stakingRevenue is value go to team ?

Reply:
Yes this is correct, the staking revenue is part of the protocol rev, so should be in dailyRevenue and also in dailyProtocolRevenue